test/container_app_test.py

```diff
@@ -19,6 +19,9 @@
 def my_f_1(x):
     pass
 
+# Simulate calling `breakpoint()` in test
+def test_breakpoint():
+    return True
```

Comment:
I think the closest we can get is something like this

```suggestion
    assert True
```

Reply:
Oh I see. That's alright. We don't have to add the `assert` here. This is good to merge.